app/views/DutyDefermentRequestedStatements.scala.html

```diff
@@ -24,7 +24,11 @@
 @this(
     layout: Layout,
     missing_documents_guidance: components.missing_documents_guidance,
-    duty_deferment_file: components.duty_deferment_file
+    duty_deferment_file: components.duty_deferment_file,
+    h1: components.h1,
+    h2: components.h2,
+    p: components.p,
+    p2: components.p2
```

Comment:
Not sure I'm being blind but can't see where this p2 is being used.

Reply:
Yep - needs removing.